### ISSUE TITLE
Synology - Outbound Connections - Cant Access From LAN

### ISSUE THREAD

**Issue by gregg098** [tailscale/tailscale#3792]
### What is the issue?

Synology DS-1520+ (Intel CPU) with DSM 7. Running 1.20.2 Synology package.

I've had this issue with the last few releases as well and the same thing with the new 1.20.2 release. If I follow the instructions from this page:
https://tailscale.com/kb/1152/synology-outbound/
Everything is OK until I restart Tailscale from the package manager. I stop it, then start it, then the web UI becomes unusable. I can still connect via the Synology's Tailscale IP however. This applies to all Synology services connectable from LAN. Only a reboot fixes it.

### Steps to reproduce

- Install latest release from Synology releases page @ https://pkgs.tailscale.com/stable/#spks
- Follow instructions from https://tailscale.com/kb/1152/synology-outbound/
- After restarting Tailscale, cannot access any Synology services from LAN IP, only from Tailscale IP.

### Are there any recent changes that introduced the issue?

I tried this starting with the 1.18.x releases I believe, and still having the issue on 1.20.2.

### OS

Synology

### OS version

DSM7

### Tailscale version

1.20.2

### Bug report

BUG-0591c9aac2ec4b10f0cb5f584aa4b178412f5c25b4328588258fdd36194bb886-20220123165639Z-8baa2e3421896c53

**Comment by bradfitz** [comment]
Dup of #3761?

**Comment by gregg098** [comment]
Don't think so. In my case, after I follow the instructions for outbound connections and restart Tailscale, I can't access anything via the Synology's IP address, only over Tailscale IP, until I reboot the device.

**Comment by DentonGentry** [comment]
Is there a subnet router advertising the Synology's LAN subnet? https://github.com/tailscale/tailscale/issues/1344 and https://github.com/tailscale/tailscale/issues/2590 might be relevant.

**Comment by gregg098** [comment]
Yes, there was (is) subnet routing on a different device for the Synology's subnet. It looks like when I submitted this bug report last year, I was on the latest version of Tailscale at the time (1.20.x).  The knowledge base article references 1.22.x or greater and has different instructions now. I would guess that the issue was resolved or was a configuration error on my part. Unfortunately, I'm unable to test on this device right now other than saying that as a basic Tailscale node with 1.36.x, it works just fine.
